carina-utils/src/test/resources/config.properties

```diff
@@ -21,6 +21,6 @@ time_format=HH:mm:ss
 
 retry_count=3
 
-test_naming_pattern={method_name}
+test_naming_pattern={test_name} {test_class} - {method_name} {method_priority} {method_thread_pool_size} {method_description}
```

Comment:
Why default pattern was changed?

Reply:
potential regression for those who used custom test_naming_pattern. erlier we inject only in the middle customized stuff but as of now whole naming might be managed via this property